src/Scheduler.php

```diff
@@ -43,6 +45,16 @@ protected function _scheduleStaticJobs(): SchedulerInterface
         $schedulerCache = $this->_getSchedulerCache();
         $this->_getSchedulerJobCollection()->setReferenceDateTime($this->_getTime()->getNow());
         foreach ($this->_getSchedulerJobTypeCollection()->getIterator() as $jobType) {
+            if (!$jobType->getIsEnabled()) {
+                $this->_getLogger()->warning(
```

Comment:
I like that we are considering this an "unhealthy" state - i.e. Job Types should only be disabled temporarily and either become deleted or re-enabled. 👌👌👌